RecallActivity.py

```diff
@@ -65,7 +65,7 @@ def __init__(self, handle):
         canvas.show()
         self.show_all()
 
-        self._game = Game(canvas, parent=self, path=self.path,
+        self._game = Game(canvas, get_activity_root(), parent=self, path=self.path,
```

Comment:
This differs from the existing code style, methods aren't usually imported from the sugar-3 API and then passed as an argument, you can easily import `get_activity_root()` in the needed file and use it there rather than import it here and pass it as an argument.

Reply:
Sure, I was not sure about this. I will change it.